ibm/service/vpc/resource_ibm_is_backup_policy_test.go

```diff
@@ -116,6 +169,62 @@ func testAccCheckIBMIsBackupPolicyConfigBasic(backupPolicyName string, vpcname,
 	}`, vpcname, subnetname, acc.ISZoneName, acc.ISCIDR, sshname, volName, acc.ISZoneName, name, acc.IsImage, acc.InstanceProfileName, acc.ISZoneName, backupPolicyName)
 }
 
+func testAccCheckIBMIsBackupPolicyConfigRemoteCopyPolicies(backupPolicyName string, vpcname, subnetname, sshname, volName, name string) string {
+	return fmt.Sprintf(`
+	resource "ibm_is_vpc" "testacc_vpc" {
+		name = "%s"
+	  }
+
+	  resource "ibm_is_subnet" "testacc_subnet" {
+		name            = "%s"
+		vpc             = ibm_is_vpc.testacc_vpc.id
+		zone            = "%s"
+		ipv4_cidr_block = "%s"
+	  }
+
+	  resource "ibm_is_ssh_key" "testacc_sshkey" {
+		name       = "%s"
+		// public_key = file("../../test-fixtures/.ssh/id_rsa")
+		public_key = file("~/.ssh/id_rsa.pub")
+	  }
+
+	  resource "ibm_is_volume" "storage" {
+		name    = "%s"
+		profile = "10iops-tier"
+		zone    = "%s"
+		# capacity= 200
+		tags 	= ["tag-0"]
+	  }
+
+	  resource "ibm_is_instance" "testacc_instance" {
```

Comment:
Do we need this instance here as backup policy is applied to volume resourcetype and their tags currently ?

Reply:
removing the unwanted instance